src/js/components/VoterGuide/VoterGuideDisplayForList.jsx

```diff
@@ -30,28 +30,28 @@ class VoterGuideDisplayForList extends PureComponent {
     const position = this.props;
 
     const {
```

Comment:
This destructuring from props can be done at the beginning of the render, and then `this.props` can be removed from the start of the variable names.